client/src/components/reply.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const Reply = (props) => {
+
+  //console.log(props)
+  if(props.replies) {
+    return (
+
+    <div className="reviewReplies" >Replies: {props.replies}</div>
+
+    )
+  } else {
+    return (<div></div>)
+  }

```

Comment:
I don't think I've used conditional rendering like this, but I'll have to try it out when I can.